notebooks/SMAP/2.0 Read and Plot SMAP data.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    group_id=list(f.keys())[1];# &lt; Lets focus on the AM overpass for this example
h5py file objects are dict-like so you should be able to access the group with `f["Soil_Moisture_Retrieval_Data_AM
</pre><p>"] </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>